packages/components/src/components/route-with-subroutes/route-with-subroutes.tsx

```diff
@@ -74,7 +91,7 @@ const RouteWithSubRoutes = ({
             );
         }
 
-        const title = getTitle?.() || '';
+        const title = getTitle ? getTitle() : '';
```

Comment:
will this work ?
const title = getTitle ()??  '';

Reply:
yes. Will incorporate it